test/OrchardCore.Tests/Apis/GraphQL/Queries/PredicateQueryTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using OrchardCore.ContentManagement.GraphQL.Queries;
+using OrchardCore.ContentManagement.GraphQL.Queries.Predicates;
+using YesSql.Indexes;
+
+namespace OrchardCore.Tests.Apis.GraphQL;
+
+public class PredicateQueryTests
+{
+    private YesSql.IConfiguration _configuration;
+
+    public PredicateQueryTests()
+    {
+        _configuration = new Configuration()
+            .UseSqlServer("Server=.;Database=OrchardCore;", "TenantSchema")
```

Comment:
Where does this connection string come from?

Reply:
This connection string is just a placeholder to instantiate the configuration object, allowing us to test our function against YesSql methods. This approach might be more effective than using mocks. I would appreciate your thoughts on this

Reply:
> This connection string is just a placeholder to instantiate the configuration object

I remember there's a connection validator in case the connection is invalid, that's why I'm asking and wondering the tests are pass :)

Reply:
It will just validate if the connection string is present, and of course, it will pass, the same approach is used in YesSql tests.

Reply:
```suggestion
            .UseSqlServer("Fake database connection string for testing;", "TenantSchema")
```

It is recommended to use an explicit error link string instead, otherwise other people may be bothered by it 😅.